tests/PKSim.Tests/Core/IndividualSimulationEngineSpecs.cs

```diff
@@ -56,7 +56,7 @@ protected override async Task Context()
          _simulation = A.Fake<IndividualSimulation>();
          _simulation.Name = "Hello";
          _simulation.DataRepository = new DataRepository();
-         _simulation.CompoundPKFor("TOTO").AucIV = 55;
+         _simulation.CompoundPKFor("TOTO").BioAvailabilityFor(0).AucIV = 55;
```

Comment:
I don't understand why we pass the 0 here. 
I would expect 
 _simulation.CompoundPKFor("TOTO").BioAvailability = 55;

Reply:
These tests need attention. They don't work so there will be another commit coming.
In general though, for the individual simulations we don't need to keep track of the actual BioAvailability parameter. We just need to hold on to AucIV because we always calculate AucInf when we open the simulation editor

Reply:
oh, and 0 was for the 'IndividualId'. This is just another case of changing everything and then cleaning up after.